apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue

```diff
@@ -104,24 +127,42 @@ const handleClickEdit = () => {
     state.isEditMode = true;
 };
 
-const handleEditCancel = () => {
-    state.isEditMode = false;
-    // TODO: Implement cancel
+const handleUpdateIsVersionValid = (isValid: boolean) => {
+    state.isVersionValid = isValid;
+};
+const handleUpdateIsTagsValid = (isValid: boolean) => {
+    state.isTagsValid = isValid;
 };
```

Comment:
suggestion: If there're only updating values, wouldn't it be better to use `sync`?🧐

Reply:
It couldn't because It shouldn't change state to props!